sensor_control/battery_monitor/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(battery_monitor)
```

Comment:
Would have preferred navigator_battery_monitor to keep with the navigator_* theme